ons_alpha/jinja2/templates/base.html

```diff
@@ -67,6 +278,32 @@
     {% endif %}
 {% endblock %}
 
+{% block header %}
+    {{
+    onsHeaderNew({
+    "wide": pageConfig.wide,
+    "fullWidth": pageConfig.fullWidth,
```

Comment:
Indentation on this section looks a little unclear

Reply:
I think this was the linter's doing unfortunately, but I'll check again.

Reply:
Nah I can't get it to let me indent it nicely. sorry.